nuxeo-features/nuxeo-platform-task/nuxeo-platform-task-core/src/test/java/org/nuxeo/ecm/platform/task/core/TaskServiceTest.java

```diff
@@ -564,12 +559,13 @@ public void testPrefixedUnprefixedActorNames() throws Exception {
         List<String> actors = new ArrayList<>();
         actors.add(user1.getName());
         actors.add(NuxeoPrincipal.PREFIX + user2.getName());
-        actors.add(SecurityConstants.ADMINISTRATORS);
-        actors.add(NuxeoGroup.PREFIX + SecurityConstants.MEMBERS);
+        actors.add("administrators");
```

Comment:
i think it can replaced by the use of `List.of` java 11, if it's possible as you did it below `List.of(user1.getName())`, thx

Reply:
Indeed. And there is also other case of an Immutable list.
I'll change it. Thanks!

Reply:
Cool, Thx ;)